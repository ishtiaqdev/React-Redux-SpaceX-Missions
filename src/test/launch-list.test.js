import React from 'react';
import { configure, mount, shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import LaunchList from '../components/launch-list';
import store from "../store";

configure({ adapter: new Adapter() });

describe('Launch List should render correctly with redux store', () => {
    it('should render correctly with redux store in "debug" mode', () => {
      const launchStore = store.getState().launchStore;
      const component = mount(<LaunchList debug launches={launchStore.launches} isSearched={launchStore.isSearched} errors={launchStore.errors} />)
    
      expect(component).toMatchSnapshot();
    });
});

//Further test can be added here.
