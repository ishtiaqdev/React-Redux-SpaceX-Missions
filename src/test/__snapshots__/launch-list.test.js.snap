// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Launch List should render correctly with redux store should render correctly with redux store in "debug" mode 1`] = `ReactWrapper {}`;
